nominal/core/workbook.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Protocol
+
+from nominal._api.combined import (
+    scout,
+    scout_notebook_api
+)
+from nominal.core._clientsbunch import HasAuthHeader
+from nominal.core._utils import HasRid
+
+
+@dataclass(frozen=True)
+class Workbook(HasRid):
+    rid: str
+    charts: any
```

Comment:
`any` is a built-in function, not the type `typing.Any` - but also, we shouldn't be using `Any`

Reply:
Sorry, the intent was just that another developer could pick it up and type correctly! (seems like we went a different direction anyway)